css/styles.css

```diff
@@ -181,6 +182,7 @@ body {
 
 .projects-inner {
   display: flex;
+  flex-flow: row wrap;
```

Comment:
What's the affect here?

Reply:
The `-webkit` version of this rule is on the line below, this adds the default rule to match with it. Linter caught this and complained